---
float64 dist
float64 avg_vel
